contracts/asset-proxy/package.json

```diff
@@ -29,10 +29,12 @@
         "profiler:report:html": "istanbul report html && open coverage/index.html",
         "coverage:report:lcov": "istanbul report lcov",
         "test:circleci": "yarn test",
+        "contracts:gen": "contracts-gen",
         "lint-contracts": "solhint -c ../.solhint.json contracts/**/**/**/**/*.sol"
     },
     "config": {
-        "abis": "generated-artifacts/@(ERC20Proxy|ERC721Proxy|IAssetData|IAssetProxy|IAuthorizable|MixinAuthorizable|MultiAssetProxy).json"
+        "abis": "./generated-artifacts/@(DummyERC20Token|DummyERC721Receiver|DummyERC721Token|DummyMultipleReturnERC20Token|DummyNoReturnERC20Token|ERC20Proxy|ERC721Proxy|IAssetData|IAssetProxy|IAuthorizable|MixinAuthorizable|MultiAssetProxy).json",
+        "abis:comment": "This list is auto-generated by contracts-gen. Don't edit manually."
```

Comment:
Let's put this _above_ the "abis" entry. Comments are always _above_ the line they comment on in our code. Less likely someone doesn't see it.

Reply:
I'm affraid I can't do that with a current architecture. When I contruct an object - the order doesn't matter.
If I put it like that:
```
packageJSON = {
        ...packageJSON,
        config: {
            ...packageJSON.config,
            'abis:comment': AUTO_GENERATED_BANNER_FOR_LISTS,
            abis: `${artifactsDir}/@(${sortedContractNames.join('|')}).json`,
        },
    };
```
Or like that:
```
packageJSON = {
        ...packageJSON,
        config: {
            ...packageJSON.config,
            abis: `${artifactsDir}/@(${sortedContractNames.join('|')}).json`,
            'abis:comment': AUTO_GENERATED_BANNER_FOR_LISTS,
        },
    };
```
The result is the same. The order of the keys in an object in JS is not defined and I can't influence it.